tests/api/test_types.py

```diff
@@ -248,3 +254,43 @@ def test_local_manifests_load_invalid_data():
         # Valid to deserialize, invalid fields
         with pytest.raises(DataError):
             cls.decrypt_and_load(valid_msgpack_but_bad_fields, key=key)
+
+
+def test_remote_manifests_load_invalid_data(alice: LocalDevice):
+    key = SecretKey.generate()
+    valid_zip_msgpack_but_bad_fields = zlib.compress(packb({"foo": 42}))
```

Comment:
can you also add a test with
- `valid_zip_bud_bad_msgpack = zlib.compress(b"dummy")`
- `invalid_zip = b"dummy"`

those are cheap tests and cover real things ;-)

Reply:
(and if you can also add the `invalid_zip` to `test_local_manifests_load_invalid_data` too)

Reply:
done